src/components/bandCombos/BandComboSearchForm.js

```diff
@@ -46,11 +47,64 @@ class BandComboSearchForm extends Component {
   handleSubmit(e) {
     e.preventDefault();
     const { dispatch } = this.props;
+    const query = qs.stringify({
+      ...this.state,
+      colours: this.state.colours.map(colour => colour.value).join(','),
+    })
+    dispatch(getBandCombos(query));
+  }
 
-    dispatch(getBandCombos(qs.stringify(this.state)));
+  _renderMenuItemChildren(colour, props, index) {
+    const style = {
+      backgroundColor: colour.hex,
+      color: colour.hex,
+      marginRight: '.5rem',
+    };
+    if (colour.value === 'white') {
+      style.border = '1px solid #ddd';
+    }
+    return (
+      <React.Fragment key={index}>
+        <span style={ style }>colour</span>
+        <span>{ colour.label }</span>
+      </React.Fragment>
+    );
+  }
+
+  _renderToken(colour, props, index) {
+    const style = {
+      backgroundColor: colour.hex,
+      color: colour.hex,
+    };
+    if (colour.value === 'white') {
+      style.border = '1px solid #ddd';
+    }
+    return (
+      <Token
+        key={ index }
+        onRemove={ props.onRemove }
+        style={ style }
+      >
+        c
```

Comment:
@sgmbr What's the relevance of the 'c'?

Reply:
It was a placeholder for adding height to the rendered div. Changed from 'c' to 'col' to add some meaning.